rfcs/experimental/flow_graph_msg_keys/README.md

```diff
@@ -0,0 +1,226 @@
+# Type-specified message keys for join_node
+
+## Introduction
+
+This experimental extension allows a key matching `join_node` to obtain keys via functions associated with its
+input types. The extension simplifies the existing approach by removing the need to provide a function object for
```

Comment:
It seems like the first sentence describes the "production" behavior, not the experimental extension.

Reply:
I've made fixes.